rust/tool/tests/common/mod.rs

```diff
@@ -115,12 +135,15 @@ fn random_string(length: usize) -> String {
 pub fn lanzaboote_install(
     config_limit: u64,
     esp_mountpoint: &Path,
+    system_double: &str,
```

Comment:
This shouldn't be a parameter since we set the system globally. With this parameter the diff would also be considerably smaller.

Reply:
Fixed